Index: kicad-0.20141025+bzr4029/kicad/prjconfig.cpp
===================================================================
--- kicad-0.20141025+bzr4029.orig/kicad/prjconfig.cpp
+++ kicad-0.20141025+bzr4029/kicad/prjconfig.cpp
@@ -89,8 +89,7 @@ void KICAD_MANAGER_FRAME::CreateNewProje
         }
         else
         {
-            templatePath = wxPathOnly(wxStandardPaths::Get().GetExecutablePath()) +
-                sep + wxT( ".." ) + sep + wxT( "share" ) + sep + wxT( "template" ) + sep;
+            templatePath = wxPathOnly("/usr/share/kicad/template") + sep;
         }
 
         ps->AddPage( _( "System Templates" ), templatePath );
