From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Sat, 24 Feb 2018 22:10:40 +0100
Subject: fix misspelled 'Modifed' -> 'Modified'

---
 pcbnew/dimension.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pcbnew/dimension.cpp b/pcbnew/dimension.cpp
index 9a5ae13..617f75f 100644
--- a/pcbnew/dimension.cpp
+++ b/pcbnew/dimension.cpp
@@ -230,7 +230,7 @@ void DIALOG_DIMENSION_EDITOR::OnOKClick( wxCommandEvent& event )
     m_parent->Refresh();
 #endif
 
-    commit.Push( _( "Modifed dimensions properties" ) );
+    commit.Push( _( "Modified dimensions properties" ) );
     event.Skip();   // ends returning wxID_OK (default behavior)
 }
 
