From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Sat, 24 Feb 2018 21:59:38 +0100
Subject: fix misspelled 'neeeded' -> 'needed'

---
 eeschema/sch_eagle_plugin.cpp | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/eeschema/sch_eagle_plugin.cpp b/eeschema/sch_eagle_plugin.cpp
index f4f134f..57e2d0f 100644
--- a/eeschema/sch_eagle_plugin.cpp
+++ b/eeschema/sch_eagle_plugin.cpp
@@ -1808,7 +1808,7 @@ void SCH_EAGLE_PLUGIN::addBusEntries()
                                 else
                                 {
                                     SCH_MARKER* marker = new SCH_MARKER( linestart,
-                                            "Bus Entry neeeded" );
+                                            "Bus Entry needed" );
 
                                     m_currentSheet->GetScreen()->Append( marker );
                                 }
@@ -1851,7 +1851,7 @@ void SCH_EAGLE_PLUGIN::addBusEntries()
                                 else
                                 {
                                     SCH_MARKER* marker = new SCH_MARKER( linestart,
-                                            "Bus Entry neeeded" );
+                                            "Bus Entry needed" );
 
                                     m_currentSheet->GetScreen()->Append( marker );
                                 }
@@ -1899,7 +1899,7 @@ void SCH_EAGLE_PLUGIN::addBusEntries()
                                 else
                                 {
                                     SCH_MARKER* marker = new SCH_MARKER( lineend,
-                                            "Bus Entry neeeded" );
+                                            "Bus Entry needed" );
 
                                     m_currentSheet->GetScreen()->Append( marker );
                                 }
@@ -1941,7 +1941,7 @@ void SCH_EAGLE_PLUGIN::addBusEntries()
                                 else
                                 {
                                     SCH_MARKER* marker = new SCH_MARKER( lineend,
-                                            "Bus Entry neeeded" );
+                                            "Bus Entry needed" );
 
                                     m_currentSheet->GetScreen()->Append( marker );
                                 }
@@ -1988,7 +1988,7 @@ void SCH_EAGLE_PLUGIN::addBusEntries()
                                 else
                                 {
                                     SCH_MARKER* marker = new SCH_MARKER( linestart,
-                                            "Bus Entry neeeded" );
+                                            "Bus Entry needed" );
 
                                     m_currentSheet->GetScreen()->Append( marker );
                                 }
@@ -2025,7 +2025,7 @@ void SCH_EAGLE_PLUGIN::addBusEntries()
                                 else
                                 {
                                     SCH_MARKER* marker = new SCH_MARKER( linestart,
-                                            "Bus Entry neeeded" );
+                                            "Bus Entry needed" );
 
                                     m_currentSheet->GetScreen()->Append( marker );
                                 }
@@ -2069,7 +2069,7 @@ void SCH_EAGLE_PLUGIN::addBusEntries()
                                 else
                                 {
                                     SCH_MARKER* marker = new SCH_MARKER( lineend,
-                                            "Bus Entry neeeded" );
+                                            "Bus Entry needed" );
 
                                     m_currentSheet->GetScreen()->Append( marker );
                                 }
@@ -2106,7 +2106,7 @@ void SCH_EAGLE_PLUGIN::addBusEntries()
                                 else
                                 {
                                     SCH_MARKER* marker = new SCH_MARKER( lineend,
-                                            "Bus Entry neeeded" );
+                                            "Bus Entry needed" );
 
                                     m_currentSheet->GetScreen()->Append( marker );
                                 }
